source/Halibut.Tests/UsageFixture.cs

```diff
@@ -315,6 +330,39 @@ static string DownloadStringIgnoringCertificateValidation(string uri)
 #endif
         }
 
+        static IEnumerable<KeyValuePair<string, string>> GetHeadersIgnoringCertificateValidation(string uri)
+        {
+#if NET40
```

Comment:
Why not just use `HttpClient` for both cases? No need to deal with ServicePointManager static shenanigans... :)

Reply:
That would make sense...

Reply:
Unfortunately that is not that case with the latest versions, https://github.com/OctopusDeploy/OctopusClients/issues/121